### PR TITLE
let KafkaChangeFeed support multiple topics

### PR DESCRIPTION
**Pull request by czue**
:dolphin: 

this will eventually allow us to just run a single UCR pillow off of kafka rather than one pillow per topic.

still in progress (need to update the checkpoints to be able to handle multiple topics), though I think this should be totally backwards compatible.

wait for build

@snopoke cc @biyeun

### CODE REVIEW

File: corehq/apps/change_feed/consumer/feed.py [hunk 1]
```diff
@@ -57,27 +74,42 @@ def iter_changes(self, since, forever):
...
-        consumer = self._get_consumer(MIN_TIMEOUT)
-        # we have to fetch one change to populate the highwater offset
+    def get_current_offsets(self):
+        consumer = self._get_consumer(MIN_TIMEOUT, auto_offset_reset='smallest')
         try:
-            consumer.next()
-        except (ConsumerTimeout, KafkaConfigurationError, KafkaUnavailableError) as e:
+            # we have to fetch the changes to populate the highwater offsets
+            # todo: there is likely a cleaner way to do this
+            changes = list(consumer)
```
Comment (dimagimon) [inline corehq/apps/change_feed/consumer/feed.py]: F841 local variable 'changes' is assigned to but never used

File: corehq/apps/change_feed/consumer/feed.py [hunk 2]
```diff
@@ -57,27 +74,42 @@ def iter_changes(self, since, forever):
...
             # kafka seems to be having issues. log it and move on
-            logging.exception(u'Problem getting latest change form kafka for {}: {}'.format(
+            logging.exception(u'Problem getting latest offsets form kafka for {}: {}'.format(
                 self,
                 e,
             ))
             return None
-        return consumer.offsets('highwater')[(self._topic, self._partition)]
+
+        highwater_offsets = consumer.offsets('highwater')
```
Comment (snopoke) [inline corehq/apps/change_feed/consumer/feed.py]: The latest version of the kafka lib seems to have a better API for getting this: http://kafka-python.readthedocs.org/en/master/apidoc/KafkaConsumer.html#kafka.KafkaConsumer.highwater

File: corehq/apps/change_feed/consumer/feed.py [hunk 3]
```diff
@@ -17,20 +17,29 @@ class KafkaChangeFeed(ChangeFeed):
...
+        self._topics = topics
         self._group_id = group_id
         self._partition = partition
 
     def __unicode__(self):
-        return u'KafkaChangeFeed: topic: {}, group: {}'.format(self._topic, self._group_id)
+        return u'KafkaChangeFeed: topics: {}, group: {}'.format(self._topics, self._group_id)
+
+    def _get_single_topic_or_fail(self):
+        if len(self._topics != 1):
```
Comment (snopoke) [inline corehq/apps/change_feed/consumer/feed.py]: bracketing error: `if len(self._topics) != 1`

Comment (snopoke) [inline corehq/apps/change_feed/consumer/feed.py]: causing build failure

**Comment by snopoke** [comment]
:heavy_check_mark: once build passes

File: corehq/apps/change_feed/consumer/feed.py (hunk 2 above)
Comment (czue) [inline corehq/apps/change_feed/consumer/feed.py]: nice, i'll look into upgrading

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**